src/Elliptic/DiscontinuousGalerkin/DgElementArray.hpp

```diff
@@ -83,7 +83,7 @@ void DgElementArray<Metavariables, PhaseDepActionList>::allocate_array(
     const std::vector<ElementId<volume_dim>> element_ids =
         initial_element_ids(block.id(), initial_ref_levs);
     int which_proc = 0;
-    const int number_of_procs = sys::number_of_procs();
+    const int number_of_procs = sys::charm::number_of_procs();
```

Comment:
Oh, I see..  This commit should be combined with the one that's the namespace change so everything still compiles on each commit.